src/Components/Server/src/DependencyInjection/ServerSideBlazorBuilderExtensions.cs

```diff
@@ -48,11 +48,13 @@ public static IServerSideBlazorBuilder AddHubOptions(this IServerSideBlazorBuild
                 throw new ArgumentNullException(nameof(builder));
             }
 
-            if (configure != null)
+            if (configure == null)
             {
-                builder.Services.Configure<HubOptions<ComponentHub>>(configure);
+                throw new ArgumentNullException(nameof(configure));
```

Comment:
This matches the behavior of `AddCircuitOptions`. Seems very strange to call this and not have it do anything by passing in a null.

Reply:
I think this follows a specific pattern, I believe we do it this way across all asp.net core. I would suggest we keep it as is for consistency.